src/components/UserTable.tsx

```diff
@@ -159,23 +184,59 @@ const UserTable: React.FC<UserTableProps> = ({ role , userType, searchPlaceholde
     setSelectedDistrict([]);
     setSelectedBlock([]);
 
+    console.log("f", filters.status)
 
     setSelectedState(selected);
   
     if (selected[0] === "") {
-      console.log("hii")
-      setFilters({ role: role, status: "active" });
+      if(filters.status)
+      setFilters({ role: role, status: filters.status });
+    else
+    setFilters({ role: role});
+
     } else {
      const stateCodes = code?.join(",");
       setSelectedStateCode(stateCodes);
-      setFilters({ role: role, status: "active", state: stateCodes });
+      if(filters.status)
+      setFilters({ role: role, status: filters.status, state: stateCodes });
+    else
+    setFilters({ role: role, state: stateCodes });
+
     }
 
     console.log("Selected categories:", typeof code[0]);
   };
   const handleFilterChange = async (event: SelectChangeEvent) => {
     console.log(event.target.value as string);
     setSelectedFilter(event.target.value as string);
+    if(event.target.value ==="Active")
+    {
+      console.log(true)
+       setFilters(prevFilters => ({
+      ...prevFilters,
+      status: "active"
+    }));
+    }
+    else if (event.target.value==="Archived")
```

Comment:
@AkshataKatwal16 add this in the constant file

Reply:
done